addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveAlarmCommandClass.java

```diff
@@ -445,6 +496,14 @@ public Integer getValue() {
                 result.add(getSupportedMessage());
             }
         }
+        if (getVersion() > 2) {
```

Comment:
hmm have to think about how to solve this. We cant use the 'refresh==true' check because we first have to request/know the notification type before we can request what events the type supports. But agree the there should be some protection against bad devices.